.gitignore

```diff
@@ -1,4 +1,4 @@
 /composer.lock
 /vendor
 /tests/tmp
-/tests/.phpunit.result.cache
+.phpunit.result.cache
```

Comment:
Just one last question, can this be listed with a leading slash, for the sake of unification with the other lines? Would it still work for you?

Reply:
Done